go.mod

```diff
@@ -4,30 +4,68 @@ go 1.12
 
 require (
 	github.com/Nordix/simple-ipam v1.0.0
-	github.com/danielvladco/k8s-vnet v0.0.0-20200326075255-ab8adf273c88
+	github.com/danielvladco/k8s-vnet v0.1.0
 	github.com/davecgh/go-spew v1.1.1
+	github.com/dtornow/cnns-nsr v0.0.0-20200331151134-1ba61ba20075
 	github.com/fsnotify/fsnotify v1.4.7
-	github.com/golang/protobuf v1.3.2
+	github.com/gofrs/uuid v3.2.0+incompatible
+	github.com/golang/protobuf v1.3.5
 	github.com/grpc-ecosystem/grpc-opentracing v0.0.0-20180507213350-8e809c8a8645
 	github.com/ligato/vpp-agent v2.3.0+incompatible
-	github.com/networkservicemesh/networkservicemesh/controlplane/api v0.2.0
-	github.com/networkservicemesh/networkservicemesh/pkg v0.2.0
+	github.com/networkservicemesh/networkservicemesh/controlplane/api v0.3.0
```

Comment:
Most of these `go.mod` changes are unnecessary.  I think you (or the IDE) must've run go mod commands that upgrades packages rather than just `go download` to pull the dependency versions in.